db/migrations/20190923070722-create_users_table.js

```diff
@@ -9,15 +9,19 @@ module.exports = {
         allowNull: false,
         autoIncrement: true
       },
-      name: { 
+      name: {
         type: Sequelize.STRING,
         allowNull: false
       },
-      email: { 
+      email: {
         type: Sequelize.STRING,
         allowNull: false,
         unique: true
       },
+      country: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
       createdAt: {
         allowNull: false,
```

Comment:
can we have a default value here, say timestamp..

Reply:
working on it

Reply:
resolved